webplugin/js/app/mck-app.js

```diff
@@ -602,7 +603,31 @@ function ApplozicSidebox() {
     //         }
     //     })
     // };
-
+    function checkForChurnCustomer(){
+        let appId=applozic._globals.appId;
+        let uploadUrl = `http://localhost:3999/users/v2/chat/plugin/get-current-plan?appId=${appId}`
```

Comment:
update the url 
the url should be fetched according to the env in which the widget is deployed

Reply:
Template literals are not allowed
concate the strings

Reply:
use var in web sdk 
(for IE support)